week04/style.css

```
@@ -0,0 +1,96 @@
+
+
+html,body{
+    height: 99%;
+    /*왜 100%로 하면 스크롤바가 생기는지?*/
+    background-color: #c1bcea;
+}
+.top_end{
+    height: 10%;
+    /*border: 2px solid blue;*/
+}
+.background{
+    display: flex;
+    flex : 1;
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
+    /* border: 2px solid black; */
+    height: 80%;
+    justify-content: space-around;
```

1. 전체 요소에 대하여 수직 중앙 정렬을 하고 싶다면, body 태그에서 `margin: 0 auto;`를 설정합니다.
2. 자식 요소에 대하여 수작 중앙 정렬을 하고 싶다면, `text-align: center;`로 충분합니다.
```suggestion
    margin: 0 auto;
    text-align: center;
```